### ISSUE TITLE
"DRAG" Continuously on

### ISSUE THREAD

**Issue by PhlOuros** [mulaRahul/keyviz#218]
**Describe the bug**
There is a constant display of the DRAG mouse event even though there are no clicks, or drags. 

**Version**: v2.0.0.alpha

**To Reproduce**
1. Click
2. Let go of left click 
3. Move mouse

**Expected behavior**
Display Left Click and not DRAG. 

**Screenshots**

**Desktop (please complete the following information):**
 - Windows 10/11

**Additional context**

**Comment by mulaRahul** [comment]
Hey @PhlOuros, I couldn't reproduce the same with your given steps. Can you share more details like the settings you're using (screenshot will do) and if possible a screen recording of the process?

**Comment by petoncle** [comment]
Hi @mulaRahul, I've also experienced the same problem with the drag key sometimes not going away (other keys too, actually). Right now, I can systematically reproduce it with keyviz 2.0.0-alpha, Windows 11, on Firefox:
1. Press middle button: now Firefox shows the wheel cursor indicating that moving the mouse will scroll the page.
2. Release middle button.
3. Mouse the mouse: keyviz now starts showing the drag "key" as pressed. This is probably already incorrect since the middle button was released.
4. Click left button: now the wheel cursor disappear and moving the mouse does not scroll the page anymore.
5. Keyviz still shows the drag key as pressed.

There are other scenarios not involving the middle mouse button that caused the drag "key" to be stuck in keyviz but I couldn't find the exact reproduction steps yet.
